.github/workflows/release.yml

```diff
@@ -226,6 +226,7 @@ jobs:
         shell: bash
         run: |
           cd ..
+          python -c "import pyogrio; print(f'GDAL version: {pyogrio.__gdal_version__}\nGEOS version: {pyogrio.__gdal_geos_version__}')"
           python -m pytest --pyargs pyogrio.tests -v
```

Comment:
For some reason I don't understand, the pytest invocation here doesn't use the `pytest_report_header` from `conftest.py` which reports GDAL version and supported drivers etc (otherwise the explicit print here would not be needed)

Reply:
I wonder if adding the `-s` flag would help with that?  Though I thought it should show the header regardless.